### PR TITLE
ES-529 "session expired" error

### PR DESCRIPTION
**Pull request by Qeas**
Root Cause : Old session token header was not removed from request object causing the auth/login request to fail with 401 "Expired session. Please login again."
Proposed Fix : Clean Authorization token before sending auth/login request
Testing Done : manual + sanity

related go lib change: https://github.com/Nexenta/go-nexentastor/commit/bca0f2039466bf39a012ca6f40af1518d6f058c0

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -212,10 +212,6 @@ release:
 	git checkout -b ${VERSION}
 	sed -i 's/:master/:v$(VERSION)/g' deploy/kubernetes/nexentastor-csi-driver.yaml
 	docker login
-	make generate-changelog
-	make container-build
-	make container-push-remote
-	git add CHANGELOG.md
```
Comment (goldranjith) [inline Makefile:218]: Are these changes related to this bugfix?

Comment (Qeas) [reply Makefile:218]: Not related to the fix, but those steps are covered in ci and there is no need for them here

Comment (goldranjith) [reply Makefile:218]: Ideally, it should've been a separate PR. Next time..

**Review by goldranjith — APPROVED** [review]
LGTM
